VirtoCommerce.CatalogModule.Web/Scripts/blades/item-detail.js

```diff
@@ -30,6 +29,10 @@
             blade.currentEntity = blade.item;
             blade.origItem = data;
             blade.isLoading = false;
+
+            //set folderPath
```

Comment:
This line leads to forcing Item modification it is really the bug. The folder path initialization code should be moved to the ItemImageWidget controller.